### PR TITLE
Fix bump-torch-mlir workflow not fetching the correct remote

### PR DESCRIPTION
**Pull request by renxida**
skip-ci: no impact on other workflows

### CODE REVIEW

**Comment by ScottTodd** [comment]
Want to convert this to a draft? This sort of debug logging shouldn't have any value once the workflow is functional, right?

File: .github/workflows/bump_torch_mlir.yml [hunk 1]
```diff
@@ -48,12 +48,13 @@ jobs:
...
-          git fetch origin
-          if [ "${{ github.event_name }}" == "workflow_dispatch" ]; then
-            git checkout ${{ github.event.inputs.bump_target }}
+          if [ "workflow_dispatch" == "workflow_dispatch" ]; then
+            bump_target=${{ github.event.inputs.bump_target }}
           else
-            git checkout origin/main
+            bump_target=origin/main
           fi
+          git fetch ${bump_target//\// }
```
Comment (ScottTodd) [inline .github/workflows/bump_torch_mlir.yml]: Why the `//\//`?

File: .github/workflows/bump_torch_mlir.yml
Comment (ScottTodd) [inline .github/workflows/bump_torch_mlir.yml:1]: The 'Create Pull Request' step is still failing: https://github.com/iree-org/iree/actions/runs/9485869842/job/26138863826?pr=17651#step:6:53
Do you want to fix that in this PR too?

File: .github/workflows/bump_torch_mlir.yml (hunk 1 above)
Comment (renxida) [reply .github/workflows/bump_torch_mlir.yml]: substitute the `/` with a space bcs bump target is specified as remote/branch whereas fetch requires `remote branch`

Comment (renxida) [reply .github/workflows/bump_torch_mlir.yml]: should this be a comment?

Comment (ScottTodd) [reply .github/workflows/bump_torch_mlir.yml]: I'm confused... is this workflow ever going to fetch a remote other than `origin`? I guess you're trying to support branches, tags, and commit hashes?

**Comment by renxida** [comment]
giving up for now and manually bumping instead

**Pull request closed without merging**